gangplank/ocp/filer.go

```diff
@@ -177,13 +220,15 @@ func (m *minioServer) kill() {
 	_ = syscall.Kill(-m.cmd.Process.Pid, syscall.SIGTERM)
 
 	// Wait for the command to end.
-	_ = m.cmd.Wait()
+	if m.cmd != nil {
+		_ = m.cmd.Wait()
+	}
 
 	// Purge the minio files since they are used per-session.
 	if err := os.RemoveAll(filepath.Join(m.dir, ".minio.sys")); err != nil {
 		log.WithError(err).Error("failed to remove minio files")
-	}
 
+	}
```

Comment:
Minor, keep '}' in the line above, so we don't have an empty line here

Reply:
fixed :)